packages/account/src/Sections/Security/ConnectedApps/connected-apps-empty.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { CONNECTED_APPS_INFO_BULLETS } from 'Constants/connected-apps-config';
+
+const ConnectedAppsEmpty = observer(() => {
```

Comment:
Why is this separated from `ConnectedAppsInfo`? seems like majority of the code is being duplicated!